packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/sdk/test/arm-test/review/arm-networkanalytics.api.md

```diff
@@ -442,6 +449,7 @@ export class NetworkAnalyticsApi {
 // @public
 export interface NetworkAnalyticsApiOptionalParams extends ClientOptions {
     apiVersion?: string;
+    cloudSetting?: string;
```

Comment:
Is it discussed whether to use string or enum in this case?  Do we allow other values besides the ones listed in KnownAzureClouds